hack/e2e.sh

```diff
@@ -88,10 +88,11 @@ oc new-app -f dist/openshift/cincinnati.yaml \
 EOF
 )" \
   -p ENVIRONMENT_SECRETS="{}" \
+  -p REPLICAS="2" \
```

Comment:
What's the point of having two replicas in the e2e test?

Reply:
Eventually stage / prod would scale pods (so that we could have zero-downtime rollout), so e2e tests ensure this scenario is tested

Reply:
I don't expect us to require two replicas in production anytime soon. Can we leave this at 1 until we use a local-to-CI registry for tests? It seems like an unnecessary burden on Quay.

Reply:
Being resilent to OSD upgrades and other disruptions is a requirement from app-sre. Its not being enforced yet, but it would be eventually.

Having >1 pod, anti-affinity rules and PDB would eventually become obligatory

Reply:
> Having >1 pod, anti-affinity rules and PDB would eventually become obligatory

Why aren't the latter two enough to achieve what we want? I.e, why do we need to set >1 pod? My understanding is that the motivation is to prevent from downtime in case of a node-failure.

Reply:
Also, as I mentioned, I'm not against testing this in general, but I'd like to postpone until we don't hit Quay directly with all pods.

Reply:
>Why aren't the latter two enough to achieve what we want?

During node upgrade it would be taken down and the pod restarted on a different node. In order to have zero-downtime during that we need a backing pod. Anti-affinity would ensure it run on a different node (so that these are not taken down at the same time). PDB would ensure any other pod disrupting operations would keep at least one pod up (e.g. both nodes update at the same time). So anti-affinity and PDB won't do much when the app has one node

Reply:
> I'd like to postpone until we don't hit Quay directly with all pods.

We're not violating any contract when we hit Quay more often. Avoiding additional requests to Quay is certainly nice, but its not required and is outside of the scope for this PR (I'll come up with another one to improve that)

Reply:
> Avoiding additional requests to Quay is certainly nice, but its not required and is outside of the scope for this PR (I'll come up with another one to improve that)

@vrutkovs Please file a Jira to track the above comment. We can use the sprint planning to schedule this in our sprint.

Reply:
That's OTA-219, me and Trevor discussed it there and most likely we won't go that route - that'd require additional external and stateful layer with little to no benefits